crates/ruff_workspace/src/options.rs

```diff
@@ -351,17 +351,45 @@ pub struct Options {
     pub src: Option<Vec<String>>,
 
     // Global Formatting options
-    /// The line length to use when enforcing long-lines violations (like
-    /// `E501`). Must be greater than `0` and less than or equal to `320`.
+    /// The maximum [width](http://www.unicode.org/reports/tr11/#Overview) of a line when
+    /// formatting code and when enforcing long-lines violations (like
+    /// `E501`).
+    ///
+    /// Must be greater than `0` and less than or equal to `320`.
+    ///
+    /// Note: Formatted code may exceed the configured line width when it can't automatically
+    /// split the line into shorter lines, e.g. because it is a too long comment or identifier.
+    #[option(
+        default = "88",
+        value_type = "int",
+        example = r#"
+        # Allow lines to have a display width up to 120.
+        line-width = 120
+        "#
+    )]
+    pub line_width: Option<LineWidth>,
+
+    /// The maximum [display width](http://www.unicode.org/reports/tr11/#Overview) of each line when
+    /// formatting code and when enforcing long-lines violations (like
+    /// `E501`).
+    ///
+    /// Must be greater than `0` and less than or equal to `320`.
+    ///
+    /// Note: Formatted code may exceed the configured line width when it can't automatically
+    /// split the line into shorter lines, e.g. because it is a too long comment or identifier.
     #[option(
         default = "88",
         value_type = "int",
         example = r#"
-        # Allow lines to be as long as 120 characters.
+        # Allow lines to have a width up to 120.
```

Comment:
nit: for consistency
```suggestion
        # Allow lines to have a display width up to 120.
```

Reply:
I think I might actually go the other way round and use the simpler term `width` everywhere.